test/views/html/IncomeTaxSummaryViewSpec.scala

```diff
@@ -40,8 +40,10 @@ class IncomeTaxSummaryViewSpec extends TaiViewSpec {
       val view = template(vm, viewModel, appConfig)
       val doc = Jsoup.parse(view.toString())
 
-      doc must haveElementAtPathWithText("#taxCodeTerm_1_1", messages("tai.taxCode.part.announce", "K") + " K")
-      doc must haveElementWithId("taxCodeDescription_1_1")
+      doc must haveElementAtPathWithText(
+        "#employmentTaxCodeTerm_1_1",
+        messages("tai.taxCode.part.announce", "K") + " K")
```

Comment:
u expecting K twice?

Reply:
I believe the first only locates the element, and the second is the _K_ that it is checking for

Reply:
No, but this has been put together to make the test work. Taking + " K" out leaves the failure: 

```nfo] - must contain details element with tax code information *** FAILED ***
[info]   [Definition of tax code part K.] not found in elements with '#employmentTaxCodeTerm_1_1' selector:[
[info]   	Definition of tax code part K. K] (IncomeTaxSummaryViewSpec.scala:43)
```

Manual viewing shows the text renders as intended "Definition of tax code part BR. " 
Note it is also designed to be hidden:

```<span class="govuk-visually-hidden">Definition of tax code part BR. </span>```

What are your thoughts? @sergi-hmrc

Reply:
ahhh okay!! i see